write-fonts/src/tables/name.rs

```diff
@@ -14,6 +14,14 @@ impl Name {
         .try_into()
         .unwrap()
     }
+
+    fn compute_version(&self) -> u16 {
+        if self.lang_tag_record.is_some() {
+            1
+        } else {
+            0
+        }
+    }
```

Comment:
is the `if` preferable to something like 

    fn compute_version(&self) -> u16 {
        self.lang_tag_record.map_or_else(|| 0, |_| 1)
    }

Reply:
clippy also doesn't like me here, and has suggested `u16::from(self.lang_tag_record.is_some())` which seems reasonably clear.